converter/src/main/java/gov/cms/qpp/conversion/model/error/ValidationResult.java

```diff
@@ -0,0 +1,31 @@
+package gov.cms.qpp.conversion.model.error;
+
+import java.util.List;
+
+public class ValidationResult {
```

Comment:
I'm sure @clydet will ask for some class level javadoc...

Reply:
soooo how 'bout a class level javadoc comment?